### PR TITLE
Front header footer patient

### PR DESCRIPTION
**Pull request by narroarrow**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,18 +4,27 @@ import Signup from './components/Signup';
 import AdminDashboard from './components/AdminDashboard';
 import CssBaseline from '@mui/material/CssBaseline';
 import { Switch, Route, Routes } from 'react-router-dom';
+import Footer from './components/Footer';
+import Navbar from './components/Navbar';
+import DoctorProfile from './components/DoctorProfile'
+import PatientProfile from './components/PatientProfile';
```
Comment (alexbara2000) [inline src/App.js:10]: add curly brackets, ex: import {footer, navbar, ...} from './components';
Also keep it consistent with ";". For example look at line 9.

File: src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import CssBaseline from '@mui/material/CssBaseline';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import Link from '@mui/material/Link';
```
Comment (alexbara2000) [inline src/components/Footer.js:6]: same as before with curly brackets

File: src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+function Copyright() {
```
Comment (alexbara2000) [inline src/components/Footer.js:8]: For now, we can remove this

Comment (alexbara2000) [reply src/components/Footer.js:8]: I mean the full function

File: src/components/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
```
Comment (alexbara2000) [inline src/components/Navbar.js:2]: Curly brackets please

File: src/components/Navbar.js [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+  const handleCloseUserMenu = () => {
+    setAnchorElUser(null);
+  };
+
+  return (
+    <AppBar position="static">
+      <Container maxWidth="xl">
+        <Toolbar disableGutters>
+          <Typography
```
Comment (alexbara2000) [inline src/components/Navbar.js:42]: for typography, we can just have it all in one line, too much splitting.

File: src/components/Navbar.js [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+            variant="h6"
+            noWrap
+            component="div"
+            sx={{ mr: 2, display: { xs: 'none', md: 'flex' } }}
+          >
+            COVID-19 App
+          </Typography>
+
+          <Box sx={{ flexGrow: 1, display: { xs: 'flex', md: 'none' } }}>
+            <IconButton
```
Comment (alexbara2000) [inline src/components/Navbar.js:52]: same here

File: src/components/Navbar.js [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+              }}
+            >
+              {pages.map((page) => (
+                <MenuItem key={page} onClick={handleCloseNavMenu}>
+                  <Typography textAlign="center">{page}</Typography>
+                </MenuItem>
+              ))}
+            </Menu>
+          </Box>
+          <Typography
```
Comment (alexbara2000) [inline src/components/Navbar.js:87]: same as before

File: src/components/PatientProfile.js [hunk 8]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {Container, Typography, Box, Grid, Link, Checkbox, FormControlLabel, TextField, CssBaseline, Button, Avatar, Card, ListItem} from '@mui/material';
```
Comment (alexbara2000) [inline src/components/PatientProfile.js:2]: This is how the import should look

**Review by alexbara2000 — APPROVED** [review]
Content is good, so I will approve this. But take the time to fix the formatting and you can merge branch after. For next time, please tag all the issues you worked on, to better keep track of them. For now, just add them in a comment.

**Review by jongjonathan — APPROVED** [review]
Nothing to add to what Alex had said in terms of the conventions and formatting, but the actual application when it ran was really nice!

**Comment by malhimaira** [comment]
@narroarrow great work, pls make changes that Alex suggested when you can and I will accept and merge.
Also @alexbara2000 if possible, pls make coding standards as a part of our release plan for sprint 2 (good work on comments! Pls just add the relevant ones to the recording stands for release plan too)

**Pull request merged**